src/app/options.cpp

```diff
@@ -343,43 +418,86 @@ const QString& CommandLineParameterError::messageForUser() const
     return m_messageForUser;
 }
 
+QString wrapText(const QString &text, int wrapAtColumn = WRAP_AT_COLUMN, int initialIndentation = 0)
+{
+    QStringList words = text.split(' ');
+    QStringList lines = {words.first()};
+    int currentLineMaxLength = wrapAtColumn - initialIndentation;
+    
+    foreach (const QString &word, words.mid(1)) {
+        if (lines.last().length() + word.length() + 1 < currentLineMaxLength) {
+            lines.last().append(" " + word);
+        }
+        else {
+            lines.append(QString(initialIndentation, ' ') + word);
+            currentLineMaxLength = wrapAtColumn;
+        }
+    }
+    
+    return lines.join("\n");
+}
+
+QString wrapOptionText(const QString &text)
```

Comment:
Don't quite understand why we can pass `WRAP_AT_COLUMN` as default value, but can not do the same with `USAGE_TEXT_COLUMN`?